### PR TITLE
fasta and genome modified

### PR DESCRIPTION
**Pull request by 13207437722**
修改了fasta和genome的第二次遍历方法成多线程模式

### CODE REVIEW

**Comment by 13207437722** [comment]
update fasta and genome

**Comment by jhu99** [comment]
wait for more update

**Review by jhu99 — COMMENTED** [review]
search_seed(v1,v2,tset,1,j,10,seed_set_mutex); // why it's 10?
// remove unnecessary compiling files, e.g. xx.o xx.exe
// upload a test script with test data, and test command for comparison

**Comment by 13207437722** [comment]
> search_seed(v1,v2,tset,1,j,10,seed_set_mutex); // why it's 10? // remove unnecessary compiling files, e.g. xx.o xx.exe // upload a test script with test data, and test command for comparison

1.For a MITE candidate seed, the minimum length of the TIR is set to 10 bp, where 10 is the value of k. In my newly uploaded code, I have modified the parameter passing to prevent any misunderstanding.
2.I have uploaded the updates

File: GCA_001433935.1.fasta
Comment (jhu99) [inline GCA_001433935.1.fasta:1]: write a link for download this file in the readme, instead of uploading the whole data

**Pull request merged**